public/app/features/dashboard-scene/scene/LibraryVizPanel.tsx

```diff
@@ -41,16 +41,17 @@ export class LibraryVizPanel extends SceneObjectBase<LibraryVizPanelState> {
   };
 
   private async loadLibraryPanelFromPanelModel() {
-    const vizPanel = this.state.panel;
+    let vizPanel = this.state.panel;
     try {
       const libPanel = await getLibraryPanel(this.state.uid, true);
       const libPanelModel = new PanelModel(libPanel.model);
-      vizPanel.setState({
+      vizPanel = vizPanel.clone({
```

Comment:
This is interesting. I tried setting the pluginId as well, but did not do this clone. I don't quite understand why it's needed.

Reply:
Thank you for the explanation!